bokehjs/examples/anscombe/.gitignore

```diff
@@ -1 +1,2 @@
 anscombe.js
+anscombe-*.png
```

Comment:
I'm actually going to save files generated by `test_examples` in a dot directory, e.g. `.tests`. Then we can exclude all in one place and don't risk interfering with committed files.